pre-docs/applying_manifest_blocks.md

```diff
@@ -168,9 +168,7 @@ Manifest block compilation can access Hiera data that you add to your Bolt confi
 
 Following the Hiera 5 convention, the default data dir is relative to `hiera.yaml` at `$BOLTDIR/data`. For config file examples, see https://puppet.com/docs/puppet/5.5/hiera_config_yaml_5.html.
 
-If a custom data provider is used (such as `hiera-eyaml`, which allows you to encrypt your data) the gem dependencies must be available to Bolt. If using `puppet-bolt` packages and installing `hiera-eyaml`
```

Comment:
@adreyer so it looks like we should just revert to the previous language here. And you wanted essentially the same thing in the bolt_installing_modules dco.

Reply:
Yeah that sounds correct. We should target docs at users who installed with the package.

I'm ok only actually having the specific commands in one place. Installing modules is where i would put it but I'm fine with it just being on the installing bolt page and linking there from elsehwere.